### DISCUSSION TITLE
[QUESTION]How to filter some specific information and then withdraw

### DISCUSSION THREAD

**Discussion by kenneth663** [python-telegram-bot/python-telegram-bot#2711] [Q&A]
### Issue I am facing

After my robot filters out the information that meets the conditions, how to withdraw this message

### Operating System

macos

### Version of Python, python-telegram-bot & dependencies

```shell
Bot API 5.3
certifi 2021.05.30
Python 3.7.3 (default, Mar  6 2020, 22:34:30)  [Clang 11.0.3 (clang-1103.0.32.29)]
```

**Comment by Bibo-Joshi** [comment]
https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right
